enclave-tls/src/lib/tls_wrappers/null/init.c

```diff
@@ -0,0 +1,14 @@
+#include <enclave-tls/log.h>
+#include <enclave-tls/err.h>
+#include <enclave-tls/tls_wrapper.h>
```

Comment:
So it looks like err.h should be included by tls_wrapper.h, encalve_quote.h and api.h to avoid explicitly including it because it is always used by caller's code right?